examples/onnxruntime/resnet50/resnet50.py

```diff
@@ -263,12 +263,12 @@ def main():
 
     if flags.QPS:
         print("resnet50, Rate = {} QPS ".format(
-            format((((flags.batch) / (sum(latency[1:]) / len(latency[1:])))),
+            format((((flags.batch) / (sum(latency[0:]) / len(latency[0:])))),
```

Comment:
Good find. 
I assume the reason why it started with `[1:]` was that it considered first run as the warmup run. 
If you doing `--run 1` it wouldn't give you the most accurate performance numbers. 

I think it is better to do warm up runs as a separate calls.

Reply:
Yes, we needed the warmup run since there was overhead during setup. 

This is required as this step also includes the additional run. A better approach here would be to set the amount of runs to `run 1` + 1 so that you still do a warmup run, but run two runs under the hood.